presto-blackhole/src/test/java/io/prestosql/plugin/blackhole/TestBlackHoleSmoke.java

```diff
@@ -69,8 +69,14 @@ public void tearDown()
     public void testCreateSchema()
     {
         assertEquals(queryRunner.execute("SHOW SCHEMAS FROM blackhole").getRowCount(), 2);
+        assertThatQueryReturnsValue("CREATE TABLE nation as SELECT * FROM tpch.tiny.nation", 25L);
```

Comment:
This doesn't really belong to `testCreateSchema`, does it?

Reply:
hmm... I remember ending up doing this for some reason but didn't remember exactly ... it relates to the fact that we cannot cleanup new created schema in BlackholeConnector.